src/views/TradingCompetition/components/ClaimModal/index.tsx

```diff
@@ -73,19 +64,23 @@ const ClaimModal: React.FC<CompetitionProps> = ({ onDismiss, onClaimSuccess, use
           {cakeReward.toFixed(2)} CAKE
         </Heading>
         <Heading mt="16px" scale="md" mb={canClaimNFT ? '16px' : '0px'}>
-          {lazioReward.toFixed(2)} LAZIO
-        </Heading>
-        <Heading mt="16px" scale="md" mb={canClaimNFT ? '16px' : '0px'}>
-          {portoReward.toFixed(2)} PORTO
-        </Heading>
-        <Heading mt="16px" scale="md" mb={canClaimNFT ? '16px' : '0px'}>
-          {santosReward.toFixed(2)} SANTOS
+          {moboxReward.toFixed(2)} MBOX
         </Heading>
         {/* NFT */}
         {canClaimNFT ? (
           <Flex alignItems="center" flexDirection="column" width="100%">
             <ImageWrapper>
-              <Image src={NftBunnies} width={128} height={128} />
+              <Image src={MboxAllBunnies} width={128} height={128} />
```

Comment:
Is the image correct or do we need another?

Reply:
For not this is ok. We'll prepare proper image by the claim phase